cmd/dep/testdata/harness_tests/init/manifest-exists/final/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I like the "don't check a nonexistent file" plan you proposed, in which case its not needed.